Gemfile.lock

```diff
@@ -345,4 +345,4 @@ DEPENDENCIES
   will_paginate-bootstrap
 
 BUNDLED WITH
-   1.17.2
+   1.17.3
```

Comment:
Should there be a particular version of `bundler` to be used mentioned in the readme? Like we have versions mentioned for `rails`, `ruby` and `postgres`. Changing the bundler version in the gemfile might create problems for other users.

Reply:
Bundler versions hardly cause any issues.

Reply:
Except when it is major release. Like bundler 1.x and bundler 2.x